src/plugins/advanced_settings/public/management_app/settings.tsx

```diff
@@ -0,0 +1,369 @@
+          callOutTitle={i18nTexts.globalCalloutTitle}
+          callOutSubtitle={i18nTexts.globalCalloutSubtitle}
+          uiSettings={globalUiSettings}
+          {...rest}
+        />
+      ),
+    },
+  ];
+
+  const [selectedTabId, setSelectedTabId] = useState(SPACE_SETTINGS_ID);
+
+  const selectedTabContent = tabs.find((obj) => obj.id === selectedTabId)?.content;
+
+  const onSelectedTabChanged = (id: string) => {
+    setSelectedTabId(id);
+  };
+
+  const renderTabs = () => {
+    return tabs.map((tab, index) => (
+      <EuiTab
+        key={index}
+        data-test-subj={`advancedSettingsTab-${tab.id}`}
+        onClick={() => onSelectedTabChanged(tab.id)}
+        isSelected={tab.id === selectedTabId}
+        append={tab.append}
+      >
+        {tab.name}
+      </EuiTab>
+    ));
+  };
+
+  const getQuery = (queryString: string, initialQuery = false): Query => {
+    try {
+      const query = initialQuery ? getAriaName(queryString) : queryString ?? '';
+      return Query.parse(query);
+    } catch ({ message }) {
+      toasts.addWarning({
+        title: parseErrorMsg,
+        text: message,
+      });
+      return Query.parse('');
+    }
+  };
+
+  const getQueryText = (search?: string): string => {
+    const queryParams = parse(search ?? window.location.search) ?? {};
+    return (queryParams[QUERY] as string) ?? '';
+  };
+
+  const getQueryState = (search?: string, intialQuery = false): AdvancedSettingsState => {
+    const queryString = getQueryText(search);
+    const query = getQuery(queryString, intialQuery);
+    const filteredSettings = {
+      namespace: mapSettings(Query.execute(query, settings)),
+      global: mapSettings(Query.execute(query, globalSettings)),
+    };
+    const footerQueryMatched = Object.keys(filteredSettings.namespace).length > 0;
+
+    return {
+      query,
+      filteredSettings,
+      footerQueryMatched,
+    };
+  };
+
+  const onQueryChange = ({ query }: { query: Query }) => {
+    setUrlQuery(query.text);
+  };
+
+  const onFooterQueryMatchChange = (matched: boolean) => {
+    setQueryState({ ...queryState, footerQueryMatched: matched });
+  };
+
+  const PageTitle = (
+    <EuiText>
+      <h1 data-test-subj="managementSettingsTitle">{i18nTexts.advancedSettingsTitle}</h1>
+    </EuiText>
+  );
+  const PageFooter = componentRegistry.get(componentRegistry.componentType.PAGE_FOOTER_COMPONENT);
+
+  return (
+    <div>
+      <EuiFlexGroup>
+        <EuiFlexItem>{PageTitle}</EuiFlexItem>
+        <EuiFlexItem>
+          <Search
+            query={queryState.query}
+            categories={categories.global.concat(categories.namespace)}
+            onQueryChange={onQueryChange}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiTabs>{renderTabs()}</EuiTabs>
+      {selectedTabContent}
+      {selectedTabId === GLOBAL_SETTINGS_ID ? (
+        <PageFooter
+          toasts={props.toasts}
```

Comment:
Why are we displaying the footer only for the global settings?

Reply:
I thought footer was used only to render telemetry setting, which is appended to the settings. With this, it's moved to global settings.

Reply:
Oh yeah, I guess it makes sense. @afharo / @Bamieh no objection?

Reply:
all good here :)

Can't wait for the day telemetry will be a simple setting 😇